src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```
@@ -0,0 +1,114 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        Storage.people.clear();
+    }
+
+    @Test
+    void password_atLeastSixSymbols_ok() {
+        Assertions.assertDoesNotThrow(() -> {
+            registrationService.register(new User("login123","ValidPassword",19));
+            registrationService.register(new User("login124","ValidP",21));
+        });
+    }
+
+    @Test
+    void login_atLeastSixSymbols_ok() {
+        Assertions.assertDoesNotThrow(() -> {
+            registrationService.register(new User("validLogin","123456",21));
+            registrationService.register(new User("validL","123456754",25));
```

According to the checklist, you should test edge cases. The minimum login length is 6. Add a test case where the login length is exactly 5, which should throw an exception. Also, add a test case where the login length is exactly 6, which should pass without an exception.